src/index.js

```diff
@@ -126,6 +128,28 @@ Object.assign(
 )
 
 /*
+ * This creates a glamorousComponentFactory for every jsxstyle element so you
+ * can simply do:
+ * const GreenBlock = glamorous.block({
+ *   backgroundColor: 'green',
+ *   padding: 20,
+ * })
+ * <GreenBlock>Click Me!</GreenBlock>
+ */
+Object.assign(
+  glamorous,
+  jsxStyleComponentTags.reduce(
+    (getters, tag) => {
+      const capitalTag = capitalize(tag)
```

Comment:
I think this is actually capitalizing the tag which we wouldn't want to do in this case. However, I'm unsure I want to support `glamorous.block` and `glamorous.flex` etc... Hmmm... What do you think?